### DISCUSSION TITLE
How does WG easy get its stats for the home page ?

### DISCUSSION THREAD

**Discussion by roycw** [wg-easy/wg-easy#1173] [Q&A]
Can any one directl me to the code wheer wireguard easy is able to egt the stats it shows ?
im interested in the RXTX stats and i assume its coming from wireguard API ? anything would be helpful

**Comment by eandersons** [comment]
Connection statistics are retrieved here utilizing `wg show`:

https://github.com/wg-easy/wg-easy/blob/62ea932d33f5299e2bd0ab742412aac7862a2254/src/lib/WireGuard.js#L158-L186
